zarr/util.py

```diff
@@ -428,7 +428,14 @@ def tree_widget_sublist(node, root=False, expand=False):
 
 
 def tree_widget(group, expand, level):
-    import ipytree
+    try:
+        import ipytree
+    except ImportError as error:
+        raise ImportError(
+            "{}: Run `pip install zarr[jupyter]` to get the required ipytree "
```

Comment:
What about "Install `ipytree` with `conda` or `pip` ..."?

Reply:
Could do, but seems weird to tell users to install a dependency if it wasn't explicitly stated as one of our dependencies, whereas `pip install zarr[jupyter]` shows that it's an _optional_ dependency of ours. I'm not fussed and happy to change it if you'd prefer though. 😄

Reply:
Yeah the main thing I'm trying to address is to make sure that Conda users handle the install through Conda (bracket notation is not a thing for Conda) and Pip wouldn't use Conda.

Not attached to the wording if you have other thoughts on how we might clarify this optional dependency to users.

Reply:
Ah good point, I didn't think of `conda`. I'll include that in the error message. :)

Reply:
Looks good. Thanks! 😄